wiki/en/Running-a-Server.md

```diff
@@ -172,6 +172,7 @@ Show an agreement window before users can connect.  The text of the agreement to
 
 ##### `-w or --welcomemessage`
 A "welcome message" to display in the Client chat window on connect. Can be given as a string or filename, and can contain HTML.
+When a path is used, the full path must be accessible by the `jamulus` Linux user. If not accessible, the literal path (rather than its contents) will appear.
```

Comment:
```suggestion
When a path is used, the full path must be accessible by the user account running Jamulus. (On Linux and Mac, user jamulus is used by default.) If not accessible, the literal path (rather than its contents) will appear. 
```